xcat-inventory/xcclient/inventory/backend.py

```diff
@@ -0,0 +1,696 @@
+#!/usr/bin/python
+from __future__ import print_function
+import 	configparser
+import os
+import sh
```

Comment:
Is `sh` python built-in module?

Reply:
no, it should be installed with pip, mentioned this in README